### DISCUSSION TITLE
Kvantum theme is not applied to the file picker in non-qt applications (xdg-desktop-portal-lxqt)

### DISCUSSION THREAD

**Discussion by e-tho** [lxqt/lxqt#2394] [Q&A]
I recently installed _xdg-desktop-portal-lxqt_ for desktop consistency as I use _pcmanfm-qt_ on _Hyprland_. The file picker works fine on chromium, vscode and firefox but the Kvantum theme is not supported. I tried to replace `QT_QPA_PLATFORMTHEME=qt5ct` with `lxqt` and to install and configure _lxqt-config_ but nothing helps. `QT_STYLE_OVERRIDE=kvantum` is set though and the theme is perfectly supported by the file manager and all other Qt applications, flatpaks included.

**Comment by Kitten-boop** [comment]
I made it work fin my case by

- Having QT_QPA_PLATFORMTHEME=qt5ct
- Installing `lxqt-qtplugin` (im on arch linux and it may be different on other distros)
- Making `~/.config/lxqt/lxqt.conf` to have this
```
[General]
__userfile__=true
icon_follow_color_scheme=true
icon_theme=Papirus-Dark
single_click_activate=true
tool_bar_icon_size=22
tool_button_style=ToolButtonTextBesideIcon

[Palette]
base_color=#2c2c2c
highlight_color=#1b91d5
highlighted_text_color=#f1f1f1
link_color=#0064ff
link_visited_color=#73739e
text_color=#f1f1f1
tooltip_base_color=#3b3b3b
tooltip_text_color=#eff0f1
window_color=#222222
window_text_color=#eff0f1

[Qt]
font="Sans Serif,10,-1,5,400,0,0,0,0,0,0,0,0,0,0,1"
style=qt5ct-style
```
- Restarting session after changes

That way lxqt file picker followed my qt theme

**Reply by tsujan** [reply]
`qt5ct` is for Qt5, `qt6ct` shouldn't be used with LXQt (or KDE or any DE that has Qt plugins), and `qt6ct` has serious issues under Wayland.

**Comment by trust2k** [comment]
https://github.com/manjaro-sway/manjaro-sway/issues/903
https://github.com/manjaro-sway/manjaro-sway/issues/979
